dev/io.openliberty.org.jboss.resteasy.common/src/io/openliberty/restfulWS/config/ConfigImpl.java

```diff
@@ -35,8 +35,20 @@
             });
         }
         for (Object key : System.getProperties().keySet()) {
```

Comment:
Not part of your changes, but I think this for loop needs to be in an else block - otherwise, an exception may be thrown if Java 2 security is enabled.

Reply:
Good catch